Sources/TemplateKit/Tag/Dump.swift

```diff
@@ -0,0 +1,18 @@
+/// Print debug information about variables
+///
+///     dump(<item>)
+///
+public final class Dump: TagRenderer {
+    public init() {}
+
+    public func render(tag: TagContext) throws -> EventLoopFuture<TemplateData> {
+        try tag.requireParameterCount(1)
+
+        var dumpString = ""
+        debugPrint(tag.parameters[0], to: &dumpString)
+
+        return Future.map(on: tag) {
```

Comment:
Using these new convenience methods are a bit nicer.
```suggestion
        return tag.future(.string(dumpString))
        return Future.map(on: tag) {
```